Dalamud/Interface/Internal/Windows/Data/Widgets/IconBrowserWidget.cs

```diff
@@ -0,0 +1,216 @@
+﻿using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Numerics;
+
+using Dalamud.Data;
+using Dalamud.Interface.Utility;
+using Dalamud.Utility;
+using ImGuiNET;
+
+namespace Dalamud.Interface.Internal.Windows.Data.Widgets;
+
+/// <summary>
+/// Data widget for browsing in-game icons.
+/// </summary>
+public class IconBrowserWidget : IDataWindowWidget
+{
+    // Remove range 170,000 -> 180,000 by default, this specific range causes exceptions.
+    private readonly HashSet<int> nullValues = Enumerable.Range(170000, 10000).ToHashSet(); 
```

Comment:
Was this reported with Lumina?

Reply:
TEXM would throw the exceptions, I checked xivapi and that range is missing there as well.

Reply:
Sorry my last message was inaccurate.

```
14:19:43.737 ERR | [TEXM] Could not load texture from "ui/icon/170000/en/170000_hr1.tex"
14:19:43.737 ERR | System.IO.FileNotFoundException: The file located at 0x5ab45c00 is empty.
14:19:43.737 ERR |    at Lumina.Data.SqPackStream.ReadFile[T](Int64 offset)
14:19:43.737 ERR |    at Lumina.Data.SqPack.ReadFile[T](Int64 offset)
14:19:43.737 ERR |    at Lumina.Data.Category.GetFile[T](ParsedFilePath path)
14:19:43.737 ERR |    at Lumina.Data.Repository.GetFile[T](Byte cat, ParsedFilePath path)
14:19:43.737 ERR |    at Dalamud.Interface.Internal.TextureManager.GetInfo(String path, Boolean rethrow) in C:\goatsoft\companysecrets\dalamud\Interface\Internal\TextureManager.cs:line 386
14:19:43.737 ERR | [WindowSystem] Error during Draw(): IconWindow
14:19:43.737 ERR | System.IO.FileNotFoundException: The file located at 0x5ab45c00 is empty.
14:19:43.737 ERR |    at Lumina.Data.SqPackStream.ReadFile[T](Int64 offset)
14:19:43.737 ERR |    at Lumina.Data.SqPack.ReadFile[T](Int64 offset)
14:19:43.737 ERR |    at Lumina.Data.Category.GetFile[T](ParsedFilePath path)
14:19:43.737 ERR |    at Lumina.Data.Repository.GetFile[T](Byte cat, ParsedFilePath path)
14:19:43.737 ERR |    at Dalamud.Interface.Internal.TextureManager.GetInfo(String path, Boolean rethrow) in C:\goatsoft\companysecrets\dalamud\Interface\Internal\TextureManager.cs:line 386
14:19:43.737 ERR |    at Dalamud.Interface.Internal.TextureManager.CreateWrap(String path) in C:\goatsoft\companysecrets\dalamud\Interface\Internal\TextureManager.cs:line 409
14:19:43.737 ERR |    at Mappy.IconWindow.DrawIcon(Int32 iconId) in D:\Documents\Visual Studio 2022\Repositories\InternalPlugins\Mappy\Mappy\MappyPlugin.cs:line 85
14:19:43.737 ERR |    at Dalamud.Interface.Utility.ImGuiClip.ClippedDraw[T](IReadOnlyList`1 data, Action`1 draw, Int32 itemsPerLine, Single lineHeight) in C:\goatsoft\companysecrets\dalamud\Interface\Utility\ImGuiClip.cs:line 109
14:19:43.737 ERR |    at Mappy.IconWindow.Draw() in D:\Documents\Visual Studio 2022\Repositories\InternalPlugins\Mappy\Mappy\MappyPlugin.cs:line 68
14:19:43.737 ERR |    at Dalamud.Interface.Windowing.Window.DrawInternal(DalamudConfiguration configuration) in C:\goatsoft\companysecrets\dalamud\Interface\Windowing\Window.cs:line 329
```

The exception thrown is two part, I was able to catch the second part successfully by checking if the file exists first, and by try catching in draw, but the exception thrown by TEXM remained.